File: src/app/model/patient.ts
export class Patient{
    constructor(
        public id:number,
        public email:string,
        public title:string,
        public firstName:string,
        public lastName:string,
        public dob:Date,
        public contactNumber:string,
        public password:string,
        public gender:string,
        public address: string,
    ){
        this.id=id ||0;
        this.email=email||"";
        this.title= title ||"";
        this.firstName=firstName ||"";
        this.lastName=lastName ||"";
        this.dob=dob ||new Date;
        this.contactNumber=contactNumber ||"";
        this.password=password ||"";
        this.gender= gender ||"";
        this.address=address ||"";
        
    }
    
}

File: authentication-service/src/main/java/com/revature/authenticationservice/model/Patient.java
package com.revature.authenticationservice.model;

import java.util.ArrayList;
import java.util.List;

import jakarta.persistence.Column;
import jakarta.persistence.ElementCollection;
import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Entity
@NoArgsConstructor
@AllArgsConstructor
@Getter
@Setter
@Builder
@Table(name = "patient")
public class Patient {
	
	@Id()
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	int id;
	@Column(unique=true)
	String email;
	String title;
	String firstName;
	String lastName;
	String dob;
	String contactNumber;
	String password;
	char gender;
	String address;
	

}
